File: library/src/main/java/cn/wandersnail/http/upload/UploadProgressListener.java
package cn.wandersnail.http.upload;

import androidx.annotation.NonNull;

/**
 * date: 2019/8/23 18:12
 * author: zengfansheng
 */
public interface UploadProgressListener {

    /**
     * 进度更新
     *
     * @param fileInfo 文件信息
     * @param progress 已完成的大小
     * @param max      总大小
     */
    void onProgress(@NonNull FileInfo fileInfo, long progress, long max);
}

File: library/src/main/java/com/snail/network/upload/UploadProgressListener.kt
package com.snail.network.upload

/**
 *
 *
 * date: 2019/5/15 11:16
 * author: zengfansheng
 */
interface UploadProgressListener {
    /**
     * 进度更新
     *
     * @param name 文件参数名
     * @param progress 已完成的大小
     * @param max 总大小
     */
    fun onProgress(name: String, progress: Long, max: Long)
}